ClinGen/clincoded
# Connects to #1600. Canonical transcript for variant title.

**Technical notes:**
The sample CA IDs that can be used for testing are **CA5123092**, **CA387747130**, **CA14274637** (intergenic), **CA352991** (overlapping genes).

**Parts of the interfaces affected by the changes:**
1. Dashboard page
2. All Interpretation page
3. Variant header on the Gene-Disease Record page
4. Variant palette on the Gene-Disease Record page
5. Family curation page
6. Family curation view-only page
7. Individual curation page
8. Individual curation view-only page
9. Experimental curation page
10. Experimental curation view-only page
11. GCI variant curation page
12. GCI evidence summary page
13. VCI header
14. VCI evaluation summary page

**Steps to test:**
1. In the VCI, enter the **CA5123092** CA ID for the variant and expect to see the title in the header consisting of the canonical transcript with both the gene symbol and protein change embedded (instead of the GRCh38 NC HGVS name).
2. Mouse over the "i" icon next to the title and expect to see a tooltip being shown.
3. Start the interpretation for this variant and evaluate some criteria to get a calculated pathogenicity.
4. Go to the summary page and provision/approve the saved interpretation.
5. In the snapshot, click on the view summary button and bring up the interpretation evaluation summary page.
6. Expect to see the variant title consisting of the canonical transcript also appearing on the summary.
7. Go to the Dashboard page and expect to see the same variant title being shown for the interpretation in question.
8. Go to the all Interpretation page and expect to see the same variant title being shown for the interpretation in question.
9. Start a new Gene-Disease record and curate a Proband evidence.
10. Enter **CA387747130** for one of the variants in the Proband evidence and expect to see the canonical transcript being used in the variant title.
11. Then enter **CA352991** for the other variant in the Proband evidence and expect to see the GRCh38 HGVS name being used in the variant title.
12. Save the curation form and return to the Gene-Disease record page.
13. Expect to see the same variant titles being shown for the 2 selected variants in the record page header as well as in the **Associated Variants** palette.
14. Click on the **View/Score** link for the Proband evidence in the curation palette and go to the view-only page for the evidence.
15. Expect to see the same variant titles being shown for the 2 selected variants.
